README.md

```diff
@@ -69,9 +76,9 @@ kubectl delete crds greptimedbclusters.greptime.io
 
 ## List of Charts
 
-- [greptimedb](./charts/greptimedb/README.md)
 - [greptimedb-operator](./charts/greptimedb-operator/README.md)
 - [greptimedb-etcd](./charts/greptimedb-etcd/README.md)
+- [greptimedb](./charts/greptimedb/README.md)
```

Comment:
why change the order? looks like unnecessary modification.

Reply:
This deploy order looks like operator->etcd->greptimedb